package/harvester-repo/Dockerfile

```diff
@@ -1,3 +1,15 @@
-FROM nginx:stable
+FROM registry.suse.com/bci/bci-base:15.3
 
-COPY charts /usr/share/nginx/html/charts
+RUN zypper -n rm container-suseconnect && \
+    zypper -n in nginx && \
+    zypper -n clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* && \
+    ln -sf /dev/stdout /var/log/nginx/access.log && \
+    ln -sf /dev/stderr /var/log/nginx/error.log
+
+COPY charts /srv/www/htdocs/charts
+
+EXPOSE 80
+
+STOPSIGNAL SIGQUIT
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
Reference for reviewers: https://github.com/nginxinc/docker-nginx/blob/b0e153a1b644ca8b2bd378b14913fff316e07cf2/stable/debian/Dockerfile#L110-L114